x-pack/legacy/plugins/ml/public/application/services/ml_api_service/index.d.ts

```diff
@@ -124,6 +124,11 @@ declare interface Ml {
 
   results: {
     getMaxAnomalyScore: (jobIds: string[], earliestMs: number, latestMs: number) => Promise<any>;
+    fetchPartitionFieldsValues: (
+      criteriaFields: Array<{ fieldName: string; fieldValue: any }>,
```

Comment:
For consistency, I think job ID should be passed as a separate parameter, rather than including it in `criteriaFields`.